server/node/src/main/java/org/kaaproject/kaa/server/operations/service/akka/actors/core/endpoint/global/GlobalEndpointActorMessageProcessor.java

```diff
@@ -106,6 +108,12 @@ private void processUnicastNotificationMsg(ActorContext context, ThriftUnicastNo
         });
     }
 
+    private void processEndpointConfigurationRefreshMsg(ActorContext context, ThriftEndpointConfigurationRefreshMessage thriftMsg) {
+        ThriftEndpointConfigurationRefreshMessage localMsg = new ThriftEndpointConfigurationRefreshMessage(thriftMsg);
+        localMsg.setActorClassifier(ThriftActorClassifier.LOCAL);
+        dispatchMsg(context, localMsg, clusterService::sendEndpointConfigurationRefreshMessage);
```

Comment:
please update all other methods to contain reference on appropriate clusterService method instead of lambda